src/http/AuthHttpClient.js

```diff
@@ -916,6 +917,7 @@ class AuthHttpClient {
             data: {
                 logo: logoPayload,
                 colors,
+                consentText: consentText,
```

Comment:
you can use `consentText,` just as `colors` above if the key is the same as the variable name